lib/middleware/api-session.ts

```diff
@@ -12,28 +12,6 @@ import Interface = require('../interface');
 var SESSIONSTORE = new SessionStore(conf.get('expiration'));
 
 // PUBLIC FUNCTIONS
-export function createSession(req: Interface.IOurRequest,
```

Comment:
What's the motivation for removing this?  It seems to be conflating the work done in `handleSession`, which seems to be doing quite a bit for different types of operations.